<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>

	<arg name="name" 		default="openGo" />
	<arg name="insole_freq" default="100" />
	<arg name="publish_transforms" 	default="true"/>
	<arg name="output_left" 	default="/grf_left/wrench"/>
	<arg name="output_right" 	default="/grf_right/wrench"/>
	<arg name="estimated_delay"	default="0"/>
	<node pkg="moticon_insoles" type="from_sdk.py" 	name="moticon_insoles" output="screen">
		<param name="name" 		value="$(arg name)" />
		<param name="insole_rate" 		value="$(arg insole_freq)" />
		<param name="left_cop_reference_frame" 			value="calcn_l" />
		<param name="right_cop_reference_frame" 		value="calcn_r" />
		<param name="foot_length" 				value="0.26"/>
		<param name="foot_width" 				value="0.09"/>
		<param name="estimated_delay"	value="$(arg estimated_delay)"/>
		<param name="wait_for_trigger" value="true"/>
		<remap from="left/wrench" 	to="$(arg output_left)"/>
		<remap from="right/wrench" 	to="$(arg output_right)"/>
		<param name="publish_transforms" value="$(arg publish_transforms)"/>
	</node>
	<include file="$(find moticon_insoles)/launch/imus_from_insole.launch"/>

	<!--node pkg="tf" type="static_transform_publisher" name="left_foot_static_broadcaster" args="-0.3 0 0 0 0.707 0.707 0 calcn_l left $(arg static_pub_freq)" />
	<node pkg="tf" type="static_transform_publisher" name="right_foot_static_broadcaster" args="0.3 0 0 0 0.707 0.707 0 calcn_r right $(arg static_pub_freq)" /-->

	<!-- left_foot right_foot  -->

</launch>


